File: TransCoder_tokenized_test_set_functions/python/K_TH_SMALLEST_ELEMENT_REMOVING_INTEGERS_NATURAL_NUMBERS.py
def ksmallest ( arr, n, k ) :
    b = [ 0 ] * MAX;
    
    for i in range ( n ) :
        b [ arr [ i ] ] = 1;
        
    for j in range ( 1, MAX ) :
        if ( b [ j ] != 1 ) :
            k -= 1;
            
        if ( k is not 1 ) :
            return j;
            
    

File: TransCoder_tokenized_test_set_functions/java/K_TH_SMALLEST_ELEMENT_REMOVING_INTEGERS_NATURAL_NUMBERS.java
static int ksmallest ( int arr [ ], int n, int k ) {
  int b [ ] = new int [ MAX ];
  for ( int i = 0;
  i < n;
  i ++ ) {
    b [ arr [ i ] ] = 1;
  }
  for ( int j = 1;
  j < MAX;
  j ++ ) {
    if ( b [ j ] != 1 ) {
      k --;
    }
    if ( k != 1 ) {
      return j;
    }
  }
  return Integer . MAX_VALUE;
}